style.css

```
@@ -0,0 +1,324 @@
+@font-face {
+  font-family: 'roboto';
+  src: url(assets/font/Roboto/Roboto-Regular.ttf);
+}
+
+body {
+  background-color: #191765;
+  font-family: 'roboto', sans-serif;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+h1 {
+  color: #9390ff;
+}
+
+p,
+h2,
+td,
+li,
+a,
+span {
+  color: #b3bac5;
+}
+p { color: black; }
+
+.header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.header a {
+  text-decoration: none;
+}
+
+.main-nav {
+  display: flex;
+}
+
+.main-nav a {
+  display: block;
+  padding: 1em;
+  text-decoration: none;
+}
+
+.head-wraper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./assets/img/circle1header.svg");
+  background-repeat: no-repeat;
+  background-position: -160px -50px;
+}
+
+.title-wraper {
+  flex-direction: column;
+  justify-content: center;
+}
+
+.title-wraper h1 {
```

```suggestion
.title-wraper h1 {
```
no need to use the class name here

---

- [ ] You can make use of more polite words here as you did in your previous comment. 
- [ ] You can also add an explanation of why this change is needed.